dbms/src/Databases/DatabaseOrdinary.cpp

```diff
@@ -33,59 +40,76 @@ namespace DB
 namespace ErrorCodes
 {
     extern const int CANNOT_CREATE_TABLE_FROM_METADATA;
+    extern const int CANNOT_CREATE_DICTIONARY_FROM_METADATA;
+    extern const int EMPTY_LIST_OF_COLUMNS_PASSED;
+    extern const int CANNOT_PARSE_TEXT;
+    extern const int EMPTY_LIST_OF_ATTRIBUTES_PASSED;
 }
 
 
-static constexpr size_t PRINT_MESSAGE_EACH_N_TABLES = 256;
+static constexpr size_t PRINT_MESSAGE_EACH_N_OBJECTS = 256;
 static constexpr size_t PRINT_MESSAGE_EACH_N_SECONDS = 5;
 static constexpr size_t METADATA_FILE_BUFFER_SIZE = 32768;
 
-static void loadTable(
+namespace
+{
+
+
+String createDictionaryFromAST(
+    ASTCreateQuery ast_create_query,
```

Comment:
Strange function. It looks not good to pass ASTCreateQuery by value here. Could we use dictionary_attributes_list by ref as argument instead? It also not clear why we modify temporary object and return String while function is called 'createDictionary...'

Reply:
We don't need it at all.